pkg/controller/controller.go

```diff
@@ -485,11 +485,13 @@ func (p *csiProvisioner) Provision(ctx context.Context, options controller.Provi
 		case pvcKind:
 			rc.clone = true
 		default:
-			klog.Infof("DataSource specified (%s) is not supported by the provisioner, waiting for an external data populator to create the volume", options.PVC.Spec.DataSource.Kind)
```

Comment:
This log message is removed because the returned error will result in a very similar log message being created by the caller